BWWalkthroughExampleTests/BWWalkthroughExampleTests.swift

```diff
@@ -1,36 +0,0 @@
-//
```

Comment:
You saw everything that I did because I did it whilst I was sharing my screen with you. The file still exists.